config/FpConfig.h

```diff
@@ -272,7 +273,7 @@ typedef FwIndexType FwQueueSizeType;
 
 // Specifies the size of the buffer that contains a communications packet.
 #ifndef FW_COM_BUFFER_MAX_SIZE
-#define FW_COM_BUFFER_MAX_SIZE 128  //!< Max size of Fw::Com buffer
+#define FW_COM_BUFFER_MAX_SIZE (FW_LOG_STRING_MAX_SIZE + sizeof(FwEventIdType) + sizeof(FwPacketDescriptorType))
```

Comment:
Since the max log string length tends to be the largest buffers fprime needs to send this makes logical sense, but conceptually it feels a bit odd to tie the size of this core construct to our logging string length.

Practically, it's also nice to have the com buffer size defined to be a constant because it makes it easy to check and see if you're going over the max combuffer size without needing to do any napkin math.

I might consider keeping this a constant with a note that this should be larger than the log max string size.

Reply:
Thanks, Josh, I changed it to 512. The logic of what needs to be greater than what is somewhat complicated -- I decided to leave it to the comments in the static checks.